decidim/decidim
# Use the component name in the sidebar title

#### :tophat: What? Why?

This PR introduces a helper called `component_name` to display the name of the component displayed or the default generic name of the module (normally present in the I18n translations).

#### :pushpin: Related Issues

- Fixes #11080

#### Testing

- Edit the names of the different components of a participatory space
- Visit each component, you should see the component name and not the generic name
- In staging [Decidim Fest 2022](https://decidim-redesign.populate.tools/conferences/DecidimFest22) is a good candidate to testing

### :camera: Screenshots

![Screenshot 2023-06-23 at 05 44 26](https://github.com/decidim/decidim/assets/17616/40d1dca2-28b4-4092-9fe8-340f0322a37c)


:hearts: Thank you!
